### PR TITLE
New package: buildifier-5.1.0

### PR DESCRIPTION
**Pull request by n1c00o**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl
  - x86_64-musl
  - *should support cross-compilation with Go*

### CODE REVIEW

File: srcpkgs/buildifier/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version=5.1.0
+revision=1
+wrksrc="buildtools-${version}"
+build_style=go
+go_import_path="github.com/bazelbuild/buildtools/buildifier"
+short_desc="Format bazel BUILD and .bzl files with a standard convention"
+maintainer="n1c00o <git.n1c00o@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/bazelbuild/buildtools/tree/master/buildifier"
+#changelog="https://github.com/bazelbuild/buildtools/releases"
```
Comment (meator) [inline srcpkgs/buildifier/template:12]: Either delete this comment or uncomment it and provide (preferably raw text) changelog.
```suggestion
```

File: srcpkgs/buildifier/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+distfiles="https://github.com/bazelbuild/buildtools/archive/refs/tags/${version}.tar.gz"
+checksum=e3bb0dc8b0274ea1aca75f1f8c0c835adbe589708ea89bf698069d0790701ea3
+tags="bazel build format lint"
```
Comment (meator) [inline srcpkgs/buildifier/template:15]: `tags` is seldomly used in Voidlinux packages and its usefulness is questionable. `xbps-query` can't search packages by tags, so it only appears in the package metadata and it isn't directly used by XBPS nor any utility that I would know of.

```sh
. /usr/share/xbps.d/00-repository-main.conf && wget -qO - "$repository/x86_64-repodata" | tar --zstd -xOf - index.plist | fgrep -B 40 "<key>tags"
```
This shell command searches for everything in the repository index that contains the `tags` property. This shows that the only package in the main repository with the `x86_64` architecture is `tuigreet`. This shows that only a single package in the entire Voidlinux repository is using `tags`.

**Comment by meator** [comment]
There shall be only a [single commit per package](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes). Please squash your three commits into one.

**Comment by n1c00o** [comment]
I removed the commented line and the tag line as asked

**Comment by n1c00o** [comment]
The force-push should have rebased

**Comment by tranzystorek-io** [comment]
Shouldn't we package bazel buildtools as a whole instead?

**Comment by n1c00o** [comment]
@tranzystorek-io I do have other requests to make containing `unused_deps` and `buildozer`, templates can be found on my fork (feat/* branches).
However, since they are not directly dependent on each other, it is easier and more minimal to only install the tool needed

**Comment by n1c00o** [comment]
Nevermind, I do think it would be better to package all buildtools instead of adding them one by one

If maintainers do agree, could they close this merge request so I could start a new one shipping all buildtools in a single package named `bazel-buildtools` or something?

**Comment by tranzystorek-io** [comment]
I think there's nothing stopping you from amending this amd renaming the PR 😄

**Comment by n1c00o** [comment]
The fact that I kinda destroyed my local repository history due to me not knowing how to rebase efficiently.
It would also allow me to create other merge requests for other packages without requiring this package to be merged (these changes were made in my master 🤦‍♂️)

**Comment by tranzystorek-io** [comment]
In that case you should be able to close the PR yourself

**Comment by n1c00o** [comment]
Yes I can, mb

**Pull request closed without merging**